### ISSUE TITLE
Enable form validation for non super-admin users

### ISSUE THREAD

**Issue by joopmartens** [pluginsGLPI/formcreator#206]
Currently only users provided with the super-admin profile can validate forms.
Assigning a user e.g. the observer or supervisor profile currently adds these users to the validators list to be selected at form creation though the accept/decline buttons while validating a form can't be used without super-admin access rights.

Is there any access right I need to add to a profile to allow a user to validate a form? The wiki pages regarding validation and rights are still empty and I did not find any access rights concerning forms that can be assigned to profiles. 

Thanks in advance for your support.

**Comment by joopmartens** [comment]
After upgrading to GLPI 0.90.5 and Forms plugin 0.90-1.3.4 this problem seems to be resolved. Now the "Assistance -Forms" menu item containing the list of forms to be invalidated is available and accessible for the observer and supervisor profiles.

**Comment by joopmartens** [comment]
After upgrading to GLPI 0.90.5 and Forms plugin 0.90-1.3.4 this problem still does not seem to be resolved. The accept/decline buttons while validating a form still can't be used without super-admin access rights.

**Comment by btry** [comment]
Hi

A user can validate a form if his profile has **validate a request** or **validate an incident** right. I think your users don't have one of them.

These rights are in the tab **Assistance** of profiles.

(this capture shows the validation rights for a supervisor, as found in a fresh install of GLPi)
![image](https://cloud.githubusercontent.com/assets/14139801/18511432/ef47ba28-7a83-11e6-9287-a7296aa810a1.png)

Can you check your profiles, update if necessary and feedback ?

**Comment by joopmartens** [comment]
Hi,

Thanks you for your reply.
The validation rights had been properly assigned so that was not the problem.
Though while checking this it made me think about this in more detail and I started troubleshooting the differences of access rights between the super-admin profile and the observer profile I was using.
I found out that the problem was caused by the different entities that I'm using.

The form was created in the main entity and the validator was only assigned to a sub entity.
Because super-admin has recursive access to the main entity it was working just fine for that profile.

I now have taken care of proper form/ticket/user entity alignment and that has solved the issue.

Thanks for getting me on track with your suggestion.
